torch/onnx/_internal/fx/onnxfunction_dispatcher.py

```diff
@@ -224,7 +229,7 @@ def get_aten_name(
             # overloadpacket for some reasons.
             # https://github.com/pytorch/pytorch/issues/97201
             aten_op_default = node.target.default
-            return aten_op_default.name()  # type: ignore[attr-defined]
+            return registration.OpName.from_op_overload(op_overload=aten_op_default)  # type: ignore[no-any-return]
 
         if _symint_symfloat_builtin_to_exporter_key_table(node.target) is not None:
```

Comment:
nit: walrus to assign `aten_op` and remove L254, as well as L255 for None check because we know it is not None.

Reply:
Done